packages/nimble-components/src/table-column/date-text/cell-view/index.ts

```diff
@@ -0,0 +1,57 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { template } from '../../text-base/cell-view/template';
+import type {
+    TableColumnDateTextCellRecord,
+    TableColumnDateTextColumnConfig
+} from '..';
+import { styles } from '../../text-base/cell-view/styles';
+import { TableColumnTextCellViewBase } from '../../text-base/cell-view';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-date-text-cell-view': TableColumnDateTextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying date/time fields as text
+ */
+export class TableColumnDateTextCellView extends TableColumnTextCellViewBase<
+TableColumnDateTextCellRecord,
+TableColumnDateTextColumnConfig
+> {
+    private columnConfigChanged(): void {
+        this.placeholder = this.columnConfig.placeholder;
+    }
+
+    private cellRecordChanged(): void {
+        if (typeof this.cellRecord.value === 'number') {
+            const formatter = new Intl.DateTimeFormat(undefined, {
+                dateStyle: 'medium',
+                timeStyle: 'medium'
+            });
+            try {
+                this.text = formatter
+                    .format(this.cellRecord.value)
+                    .replace('\u202f', ' '); // on Chrome, the space before AM/PM is a narrow non-breaking space. For testing consistency across browsers, replace it with a regular space.
```

Comment:
I'm not sure this is a buggy behavior worth overriding. Did you check from that perspective?

If the goal is just making tests easier it should be a part of the page object. I'm not sure there is a clear benefit in trying to override the details of the native date formatter to get the exact same rendered text across browsers. Is there a specific argument for that?

One argument against could be that many apps are likely to use the same formatter. So our date text will render similarly to most other web apps in Chrome.

Reply:
I don't think it's a bug--just a mostly trivial difference between how each browser formats the date. It seems more targeted to doing the replacement here than in the page object. If I do it in the page object, I either have to do this substitution for *all* text rendered in cells, or add an esoteric new function to fetch a rendered date from a cell. I don't see any practical downsides to replacing one narrow non-breaking space with a regular space. That was my thought process, at least.

Reply:
Seems like the date table column could have it's own page object

Reply:
> don't see any practical downsides to replacing one narrow non-breaking space with a regular space.

- It's arbitrary and not clear why it's actually needed
- Only thing it cites is improving testing, why is that leaking in product code
- The logic is duplicated poorly across multiple files
- Doesn't link to anything explaining the behavior, why it's allowed or not allowed, etc.
- No regression testing behavior for the browser-specific workaround
- No guidance or analysis of other browser-specific behaviors we need to be aware of with this api